pyne/alara.py

```diff
@@ -266,20 +266,21 @@ def record_to_geom(mesh, cell_fracs, cell_mats, geom_file, matlib_file,
 
         ve_mixture = {}
         for row in cell_fracs[cell_fracs['idx'] == i]:
-            if cell_mats[row['cell']].metadata['mat_number'] \
-                not in ve_mixture.keys():
-                ve_mixture[cell_mats[row['cell']].metadata['mat_number']] = \
-                    round(row['vol_frac'], sig_figs)
+            cell_mat = cell_mats[row['cell']]
+            name = cell_mat.metadata['name']
+            if _is_void(name):
+                name = "mat_void"
+            if name not in ve_mixture.keys():
+                ve_mixture[name] = round(row['vol_frac'], sig_figs)
```

Comment:
Not that this is blocking, per say, but should `np.round()` be used instead of the built-in `round()`